.changeset/stale-scissors-march.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': major
```

Comment:
Should be a patch change, not major. Please read https://semver.org/

```suggestion
'@itwin/itwinui-react': patch
```

Reply:
done!

Reply:
forgot to push?

Reply:
pushed now